app/src/main/AndroidManifest.xml

```diff
@@ -12,6 +12,7 @@
         android:label="@string/app_name"
         android:supportsRtl="true"
         android:theme="@style/AppTheme"
+        android:requestLegacyExternalStorage="true"
```

Comment:
This enables existing media queries to work as is and needs to added to implementing app. This is an 80/20 fix, the alternative is rewriting how we access files entirely.

Reply:
After some research, i think the fix is we simply display a single button where the files would be, and the users open a system file picker instead of ours